project/plugins.sbt

```diff
@@ -1,7 +1,9 @@
 resolvers += Resolver.url("hmrc-sbt-plugin-releases",
   url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
 
```

Comment:
This file is missing the following resolver:
```resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"```
It may still be working locally and for a long time in build because a dependency is cached but better avoid a surprise later

Reply:
Resolver Added.